src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Search/FindAddress/FindAddress.xaml

```diff
@@ -2,55 +2,23 @@
-    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls"
-    mc:Ignorable="d"
-    d:DesignHeight="300" d:DesignWidth="300">
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
     <Grid>
-        <VisualStateManager.VisualStateGroups>
-            <VisualStateGroup>
-                <!-- Adapt screen to more than 780 effective pixels -->
-                <VisualState>
-                    <VisualState.StateTriggers>
-                        <AdaptiveTrigger MinWindowWidth="720" />
-                    </VisualState.StateTriggers>
-                    <VisualState.Setters>
-                        <Setter Target="appBar.(Grid.Row)" Value="0" />
-                    </VisualState.Setters>
-                </VisualState>
-            </VisualStateGroup>
-        </VisualStateManager.VisualStateGroups>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-            <RowDefinition Height="auto" />
-        </Grid.RowDefinitions>
-        <esriUI:MapView x:Name="MyMapView" Grid.Row="1" />
-        <CommandBar x:Name="appBar" Grid.Row="2">
-            <AppBarButton Icon="Find" Label="Search" Name="MyAppBarSearchButton" IsEnabled="False" AllowFocusOnInteraction="True">
-                <AppBarButton.Flyout>
-                    <Flyout>
-                        <StackPanel Width="200">
-                            <TextBox x:Name="MySearchBox" PlaceholderText="Esri" Width="200" />
-                            <Button Content="Search" Click="SearchButton_Click" Width="200" />
-                        </StackPanel>
-                    </Flyout>
-                </AppBarButton.Flyout>
-            </AppBarButton>
-            <AppBarButton Icon="List" Label="Suggest" Name="MyAppBarSuggestButton" IsEnabled="False">
-                <AppBarButton.Flyout>
-                    <MenuFlyout>
-                        <MenuFlyout.Items>
-                            <MenuFlyoutItem Text="277 N Avenida Caballeros, Palm Springs, CA" Click="OnSuggestionChosen" />
-                            <MenuFlyoutItem Text="380 New York St, Redlands, CA 92373" Click="OnSuggestionChosen" />
-                            <MenuFlyoutItem Text="Београд" Click="OnSuggestionChosen" />
-                            <MenuFlyoutItem Text="Москва" Click="OnSuggestionChosen" />
-                            <MenuFlyoutItem Text="北京" Click="OnSuggestionChosen" />
-                        </MenuFlyout.Items>
-                    </MenuFlyout>
-                </AppBarButton.Flyout>
-            </AppBarButton>
-        </CommandBar>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border BorderBrush="Black" BorderThickness="1" Background="White"
+                HorizontalAlignment="Right" VerticalAlignment="Top"
+                Margin="30" Padding="20" MinWidth="375">
+            <StackPanel>
+                <TextBlock Text="Enter a search or select a preset."
```

Comment:
Similar to my comment on FeatureLayerQuery sample: IMO it is not immediately obvious that user has to press Enter or click [🔍] to do the search.